screens/Settings/notificationSetter.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react'
+import { Platform, StyleSheet, View, TouchableOpacity, Text } from 'react-native'
+import { useThemeColor } from '../../components/Themed'
+
+import DateTimePicker from '@react-native-community/datetimepicker'
+import Notify from './notificationHandler'
+import WeekdayPicker from '../../components/WeekdayPicker'
+import Toast from 'react-native-simple-toast'
+
+const NotificationSetter = () => {
+  const [time, setTime] = React.useState(new Date())
+  const [pickedTime, setPickedTime] = React.useState(false)
+  const [weekdays, setWeekdays] = React.useState([-1])
+  const [show, setShow] = React.useState(false)
+  const changeTimeHandler = (event: Event, value?: Date) => {
+    setShow(Platform.OS === 'ios')
+    console.log(value?.getHours(), value?.getMinutes(), value?.getSeconds())
+    if (value) {
+      setPickedTime(true)
+    } else {
+      setPickedTime(false)
+    }
+    setTime(value || new Date())
+  }
+
+  const timeString = (timeSet: Date) => {
+    const timeHours = timeSet.getHours()
+    const timeMinutes = timeSet.getMinutes()
+    const hours =
+      timeHours === 0 ? '00' : timeHours <= 9 ? '0' + timeHours.toString() : timeHours.toString()
+    const minutes =
+      timeMinutes === 0
+        ? '00'
+        : timeMinutes <= 9
+        ? '0' + timeMinutes.toString()
+        : timeMinutes.toString()
+    return hours + ' : ' + minutes
+  }
+
+  return (
+    <>
```

Comment:
Let's wrap this with a Dialog so we can avoid placing this component inside the list: https://callstack.github.io/react-native-paper/dialog.html

Reply:
Done ✔